journal/src/main/java/io/camunda/zeebe/journal/RecordDataWriter.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.journal;
+
+import org.agrona.MutableDirectBuffer;
+
+public interface RecordDataWriter {
```

Comment:
:x: I believe this is a duplicate of `BufferWriter`, so we could simply use this, unless there is a semantic reason to differentiate between either?

Reply:
nice that fits perfectly!